commands

```diff
@@ -26,7 +26,7 @@ case "$1" in
     fi
 
     mkdir -p "$SERVICE_ROOT" || dokku_log_fail "Unable to create service directory"
-    password=$(date +%s | sha256sum | base64 | head -c 16)
+    password=$(openssl rand -hex 16)
```

Comment:
This still generates a 16 char password that is valid for postgres? Can you do the same for mysql and mariadb?